### ISSUE TITLE
Adding Sphere again :) with RajawaliVR

### ISSUE THREAD

**Issue by uzman** [Rajawali/Rajawali#1447]
hi,
i add a simple sphere to RajawaliRenderer extended class. it is ok. (i checked Material section in the sample project)
but when i add sphere to RajawaliVRRenderer extended class i can't see the sphere.
should i use setPosition method? 
globe.setPosition(2, 1.5f, 0);
if i use setPosition i can't locate the desired place.

here is the code:

		try {
			getCurrentScene().setSkybox(R.drawable.posx3, R.drawable.negx3,
					R.drawable.posy3, R.drawable.negy3, R.drawable.posz3,
					R.drawable.negz3);
            Material material = new Material();
            material.addTexture(new Texture("globe", R.drawable.globe));
            material.setColorInfluence(0);
            globe = new Sphere(1, 24, 24);
            globe.setMaterial(material);
            //globe.setPosition(2, 1.5f, 0);
			getCurrentScene().addChild(globe);
			
			
		} catch (Exception e) {
			e.printStackTrace();
		}

i want to place sphere on right-upward in cube (skybox).

i mean for right-upward: the location between right-back-up images.
getCurrentScene().setSkyBox(front, right, back, left, up, down) 

thanks in advance.

**Comment by jwoolston** [comment]
The skybox cube is 700 units on each side. If you want the sphere in a corner, you will need to set it at some combination of 350 units. For the corner you want, it would be `<350, 350, -350>`

**Comment by uzman** [comment]
i tried 350, 350, -350 and some below values. but sphere is never seen.
then i tried.
globe = new Sphere(1, 24, 24);
        globe.setMaterial(material);
        globe.setPosition(2, 1.5f, 0);
        getCurrentScene().addChild(globe);
it put a big sphere on the right side.
is coordinate system is different?

![screenshot_2015-04-04-22-51-54](https://cloud.githubusercontent.com/assets/772331/6994312/d17eb50c-db1d-11e4-983b-6ed29d572847.png)

**Comment by jwoolston** [comment]
It put a sphere on the right side because thats where you told it to. It is big because you have it very near the camera.

**Comment by uzman** [comment]
no.
globe.setPosition(350, 350, -350);
doesn't put any sphere.

the sphere in screenshot is rendered with : (2, 1.5f, 0).

i think the coordinate system is different.

**Comment by uzman** [comment]
i am using this version : https://github.com/Rajawali/RajawaliVR
last commit is 9 months before.
but Rajawali framework is newer. 
is the problem related with it?

**Comment by jwoolston** [comment]
> the sphere in screenshot is rendered with : (2, 1.5f, 0).

This is what my comment was in reference to. Those coordinates are very near the camera. The coordinate system is not any different, at least not from 9 months ago. A globe with a radius of 1 unit nearly 500 units away from the camera (the euclidean distance to that corner) is likely going to be imperceptible. As a demonstration, render the skybox solid black and the sphere as white.

**Comment by uzman** [comment]
you say its very near the camera (2,1.5,0). i agree with you.
but you miss the things that i say before.

globe.setPosition(350, 350, -350);
doesn't put any sphere. even if you use  (100,100,-100) no sphere is rendered.

can you try put a sphere at 350,350,-350 with RajawaliVR ?

**Comment by jwoolston** [comment]
> but you miss the things that i say before.

I did not miss what you said before, you had already said it, and I gave a response in regards to it. I assure you it works. The problem is you do not seem to have a grasp of OpenGL and the basics of how the geometry works. if you draw an object at those coordinates (either 100s or 350s) and do nothing else, it will be outside of the cameras viewing frustum and OpenGL will remove it from drawing (it wouldnt project on the screen anyway). 

I am closing this as resolved because at this point it is clear you are not running into an issue or misunderstanding with the library, but rather the 3D environment.
